src/templates/_partials/_blog_post.twig

```diff
@@ -0,0 +1,16 @@
+<div class="pt-8 border-b border-gray-300 first:pt-0">
+  <p class="text-base leading-[19px] py-2 text-gray-700 font-semibold">
+    {{ blog.date }}
+  </p>
+  <h3 class="text-blue-700 h3">
+    {{ blog.title }}
+  </h3>
+  <p class="leading-8 text-gray-700 p">
+    {{ blog.description }}
+  </p>
+  <button class="px-8 py-4 border border-blue-700 rounded-full mt-[30px] mb-12">
```

Comment:
Use created `btn-outline` class in the button element as the base and add tweaks after, and use an anchor tag instead of button.